.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: python
+python:
+  - 2.7
+before_install:
+  - export DISPLAY=:99
+  - sh -e /etc/init.d/xvfb start
+install:
+  - pip install -r test-requirements.txt
+  - pip install -r $VIRTUAL_ENV/src/xblock-sdk/requirements/base.txt
+  - pip install -r $VIRTUAL_ENV/src/xblock-sdk/requirements/test.txt
+  - pip install -r $VIRTUAL_ENV/src/xblock/requirements.txt
```

Comment:
@nedbat Requirements files are flat.  The transitive requirements of packages listed in `requirements.txt` are not installed implicitly, unless they are specified in `install_requires` in `setup.py`.  However, only requirements available on PyPI can be implicitly installed in this way.  Since many of our requirements are installed from Github, we can only list them in `requirements.txt`, and we then have to resort to hacks like this one to install the transitive dependencies.

An alternative would be to flatten the list of recursive requirements manually, and list all of them in `requirements.txt` in this XBlock.  However, this is fragile as well, since then we have to update requirements in even more places in sync.

Uploading all versions of our packages to PyPI comes with its own set of problems, too.  I'm sure we could manage requirements in a better way than we currently do, but it isn't an easy problem.